python/packages/autogen-ext/src/autogen_ext/models/openai/config/__init__.py

```diff
@@ -79,6 +79,9 @@ class CreateArgumentsConfigModel(BaseModel):
 
 
 class BaseOpenAIClientConfigurationConfigModel(CreateArgumentsConfigModel):
+    # To allow `model_capabilities` field without triggering pydantic warnings.
+    model_config = ConfigDict(protected_namespaces=())
```

Comment:
We might want to revisit this. Just to let you know @jackgerrits since we already have a field `model`

Reply:
Aah my bad. I had missed the chat message from @jackgerrits about this being fixed by practice update. Lemme know if you want me to update it and remove this change in a follow up :-)

Reply:
My bad as well 😄 a follow up PR to update the version and remove the config dict setting would be nice.